supporting_content/attributes/discount_handling.md

```diff
@@ -0,0 +1,115 @@
+    - ISV Success
+
+- GCP
+  - Partner Level
+  - Premier Level
+
+## Negotiated discounts
+
+TBD
+
+## Promotional discounts
+
+- Free Trial
+- Promotion discounts given by sales representatives to onboard customers.
+
+## Usage-based discounts
+
+- An example of this is GCP's [Sustained use discounts](https://cloud.google.com/compute/docs/sustained-use-discounts) which are still usage-based but not associated to any commitments.
+- AWS' RI Volume discounts?
+
+## Partner discounts
+
+- Private Rate Discounts
+- Enterprise Discount Program
+- Solution Provider Program
+- RESELLER_MARGIN
+
+# Links to documentation
+
+- AWS
+  - Reserved Instances - https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-reserved-instances.html
+  - Savings Plans - https://docs.aws.amazon.com/savingsplans/latest/userguide/what-is-savings-plans.html
+  - Bundled discounts - https://aws.amazon.com/blogs/aws-cloud-financial-management/bundled-discounts-in-aws-cost-and-usage-report/
+  - AWS partner tiers - https://aws.amazon.com/partners/services-tiers/
+
+- GCP
+  - Committed use discounts - https://cloud.google.com/compute/docs/instances/committed-use-discounts-overview
+  - Resource-based CUDs - https://cloud.google.com/compute/docs/instances/signing-up-committed-use-discounts
+  - Spend-based CUDs - https://cloud.google.com/docs/cuds-spend-based
+  - Sustained use discounts - https://cloud.google.com/compute/docs/sustained-use-discounts
+  - Credit types - https://cloud.google.com/billing/docs/how-to/export-data-bigquery-tables/standard-usage#standard-usage-cost-data-schema
+  - GCP Partner Advantage - https://cloud.google.com/partners/become-a-partner
+
+- Microsoft
+  - Reservations - https://learn.microsoft.com/en-us/azure/cost-management-billing/reservations/save-compute-costs-reservations
+  - Savings Plans - https://learn.microsoft.com/en-us/azure/cost-management-billing/savings-plan/savings-plan-compute-overview
+  - Azure Hybrid Benefit - https://azure.microsoft.com/en-us/pricing/hybrid-benefit/#overview
+  - Azure Dev/Test Pricing - https://azure.microsoft.com/en-us/pricing/offers/dev-test/
+  - Microsoft Partnership - https://partner.microsoft.com/en-us/partnership/compare-programs
+
+# Discussion / Scratch space
+
+- AWS discount types include:
+  - DiscountedUsage - discounts covered by Reserved Instances
+  - SavingsPlanCoveredUsage - discounts covered by SavingsPlans
+  - EdpDiscount - Enterprise Discount Program
+  - SppDiscount - Solutions Provider Program
+  - RiVolumeDiscount
+  - BundledDiscount - discounted usage on a specific product/service based on the usage of another product/service.
+
+- GCP discount types include:
+  - DISCOUNT - credits earned after a contractual spending threshold is reached.
+  - COMMITTED_USAGE_DISCOUNT - resource-based commitment discounts.
+  - COMMITTED_USAGE_DISCOUNT_DOLLAR_BASE - Spend-based commitment discounts.
+  - SUSTAINED_USAGE_DISCOUNT - automatic discount earned for prolonged usage.
```

Comment:
```suggestion
  - SUSTAINED_USAGE_DISCOUNT - automatic discount earned for prolonged usage.
    > [!NOTE]
    > Sustained Use Discounts (SUDs) are handled as after-the-fact adjustments (aka credits) and do not apply to the Discount Handling attribute since this attribute requires that discounts be applied directly to the row they're discounting.
```